optimade/client/cli.py

```diff
@@ -119,6 +119,7 @@ def _get(
     include_providers,
     exclude_providers,
     exclude_databases,
+    **kwargs,
```

Comment:
Why is **kwargs only added to the `_get` function and not for the `get` function ?
The get function is the one users  are supposed to use.

Reply:
PS. I just read that you only intend to use the **kwargs for testing.
So this code now makes more sense.